web/src/main/scala/net/wiringbits/components/widgets/SignInForm.scala

```diff
@@ -55,6 +59,7 @@ import scala.util.{Failure, Success}
 
       val email = state.email.getOrElse("")
       val password = state.password.getOrElse("")
+      val captcha = Captcha.trusted(state.captcha.map(_.string).getOrElse(""))
```

Comment:
`trusted` should almost never be invoked outside tests

Reply:
I'm gonna fix this when #106 gets accepted

Reply:
Merged, thanks

Reply:
I think you are missing to integrate the captcha with the form